components/message-card.tsx

```diff
@@ -0,0 +1,112 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import { FaArrowUp, FaCalendar, FaPhone } from 'react-icons/fa6';
+import { MdEmail } from 'react-icons/md';
+
+import { cn } from '~/lib/utils';
+
+import { Button } from './ui';
+
+export default function messageCard({
+  className,
+  image,
+  name,
+  quote,
+  quoteBelow,
+  readMorePath,
+  locale,
+  more,
+  details,
+}: {
+  className?: string;
+  image: string;
+  name: string;
+  quote: string;
+  quoteBelow?: string;
+  readMorePath?: string;
+  more?: string;
+  locale: string;
+  details?: {
+    email: string;
+    phone: string;
+    session: string;
+  };
+}) {
+  return (
+    <article
+      className={cn(
+        'grid gap-5',
+        'grid-cols-[5rem,auto] lg:grid-cols-[min(30%,24rem),auto]',
```

Comment:
Explanation please

Reply:
The left value in the grid is for image width in both cases, 24rem is the max limit for image width above that the image looks out of proportion.